### DISCUSSION TITLE
Orga et autres gestions

### DISCUSSION THREAD

**Discussion by thingthing** [blackfoot-makers/flame-game#33] [General]
## 👋 Hello!

Bon Melvyn voulait tester les trucs de github donc je propose de commencer une discussion pour les trucs généraux (ça m'évitera de perdre le thread sur discord).

Donc ici je vous propose de parler de l'orga du projet et autres trucs de gestions de projet!

**Comment by thingthing** [comment]
Ok premier truc, je me propose de reprendre [l'issue de quentin](https://github.com/blackfoot-makers/flame-game/issues/14) vu qu'il aura pas le temps de la finir avant de partir comme ça on pourra commencer à essayer d'avoir plusieurs joueurs sur une partie.

Ou est-ce que quelqu'un avait déjà commencé à la reprendre?

**Reply by Nirsu** [reply]
Je crois que personne le n'as repris pour l'instant mais ça serait cool ouais
-> le prochain à finir sa tache ce met la dessus ?

PS: si pas les droits, vous pouvez me ping ou @thingthing

**Reply by thingthing** [reply]
Ok du coup je l'ai commencé, j'ai la sauvegarde de la game à la création, me manque l'update des infos au fur et à mesure et l'utilisation des infos dans le reste de la game.

D'ailleurs je vais créer une issue pour le menu de création/rejoindre une game aussi!

**Reply by Nirsu** [reply]
Oki dokie nice déja :)
